charger/easee.go

```diff
@@ -413,9 +413,11 @@ func (c *Easee) Enable(enable bool) error {
 
 	// resume/stop charger
 	action := easee.ChargePause
+	targetOpMode := easee.ModeAwaitingStart
 	targetCurrent := 0.0
 	if enable {
 		action = easee.ChargeResume
+		targetOpMode = easee.ModeCharging
```

Comment:
Nur zur Sicherheit: kommen wir wirklich immer in diesen Modus? Auch wenn das Auto schon voll sein sollte?

Reply:
Vermutlich nein. :(
Good catch! 👍 

Vermutlich ist Status 4 (completed) auch noch möglich. Ich probiere mal aus was passiert wenn ich das Ladelimit erreicht habe.

Reply:
Wenn Charge startet geht es von 2 über 6 nach 3.
Bei geladenem Fahrzeug findet nur der Übergang von 2 zu 6 statt.
Completed habe ich nicht gesehen. 
Die 32A werden bereits im State 6 gesetzt.

Wir können also hier einfach mehrere States (3,6,4) akzeptieren.
Theoretisch würde auch nur 6 reichen, aber je nach Timing kann es dazu kommen das es so schnell geht das wir 6 gar nicht mitbekommen. Daher explizit alle 3 States akzeptieren?

Reply:
Gerne